deploy/template.yaml

```
@@ -70,6 +65,9 @@ Conditions:
   EnableSpotArmInstance: !Equals [!Ref Environment, dev]
 
 Mappings:
+  StaticVariables:
+    Urls:
+      SupportManualURL: "https://team-manual.account.gov.uk/teams/CRI-Orange-team/supporting-cri-orange/hmrc-nino-runbook/#frontend-api-alarms"
```

I think the link is wrong

```suggestion
      SupportManualURL: "https://team-manual.account.gov.uk/teams/CRI-Orange-team/supporting-cri-orange/hmrc-nino-runbook/#frontend-alarms"
```